java-client/session/src/main/java/io/deephaven/client/impl/ObjectService.java

```diff
@@ -6,11 +6,123 @@
 import java.util.concurrent.CompletableFuture;
 
 public interface ObjectService {
+
+    /**
+     * A server object that supports fetching.
```

Comment:
What's "fetching"? (A logical unary RPC)

Reply:
It's actually even less than an unary RPC - w/ an unary RPC you can at least have a request message; this is more like an unary RPC where you can't have any request message parameters.